src/config.rs

```diff
@@ -39,11 +39,9 @@ pub enum Backend {
     Rodio,
 }
 
-const DEFAULT_BACKEND: Backend = if cfg!(target_os = "macos") {
-    Backend::PortAudio
-} else {
-    Backend::Alsa
-};
+fn default_backend() -> Backend {
+    return Backend::from_str(BACKEND_VALUES.first().unwrap()).unwrap_or(Backend::Alsa);
```

Comment:
To prevent the first `.unwrap()` from panicking at runtime, maybe we could add something like

```rust
#[cfg(not(any(feature = "pulseaudio_backend", feature = "portaudio_backend", feature = "alsa_backend", feature = "rodio_backend")))]
compile_error!("At least one of the backend features is required!");
```

The second `.unwrap_or(Backend::Alsa)` can IMO be converted to a simple `.unwrap`, since this should never fail.